packages/legacy/core/App/services/keychain.ts

```diff
@@ -95,6 +103,19 @@ export const loadWalletSalt = async (): Promise<WalletSalt | undefined> => {
   return JSON.parse(result.password) as WalletSalt
 }
 
+export const loadLoginAttempt = async (): Promise<LoginAttempt | undefined> => {
```

Comment:
Can you export this method from the bifold index?

Reply:
@bryce-mcmath  I didn't understand. We don't have this method in the index

Reply:
if you go into `packages/legacy/core/App/index.ts` and import it and export it from there then other wallets can access it, like BC Wallet. Other than that it looks good